content/lang/fr/404.md

```diff
@@ -1,5 +1,5 @@
-# Error
+# Erreur
 
-Yikes, looks like that page vanished.
+Yikes, ressemble à cette page disparue.
 
-Time to head [Home](/).
+Le temps de diriger [Accueil] (/).
```

Comment:
`Il est temps de retourner vers l’accueil`

(You could actually replace Accueil with “Home”. Accueil seems kind of… old school to me, but it works. If you decide to change, then it’s: `Il est temps de retourner vers la home`